src/test/java/net/sf/jabref/importer/fileformat/InspecImportTest.java

```diff
@@ -0,0 +1,157 @@
+                inspecImp.isRecognizedFormat(inStream));
+        Assert.assertFalse("InspecImporter isRecognizedFormatFalse test failed",
+                inspecImp.isRecognizedFormat(inStream));
+
+    }
+
+    /**
+     * Test method for
+     * {@link net.sf.jabref.importer.fileformat.InspecImporter#importEntries(java.io.InputStream, net.sf.jabref.importer.OutputPrinter)}
+     * .
+     */
+    @Test
+    public void testImportEntries1() {
+
+        InputStream inStream = InspecImportTest.class.getResourceAsStream("InspecImportTest2.txt");
+        try {
+            List<BibtexEntry> entries = inspecImp.importEntries(InspecImportTest.class
+                    .getResourceAsStream("InspecImportTest2.txt"), new OutputPrinterToNull());
+            assertEquals(1, entries.size());
+            BibtexEntry entry = entries.get(0);
+
+            assertEquals("The SIS project : software reuse with a natural language approach", entry.getField("title"));
+            assertEquals("Prechelt, Lutz", entry.getField("author"));
+            assertEquals(" 1992", entry.getField("year"));
+            assertEquals("Abstrakt", entry.getField("abstract"));
+            assertEquals("key", entry.getField("keywords"));
+            assertEquals("10000", entry.getField("journal"));
+            assertEquals("20", entry.getField("pages"));
+            assertEquals(" 19", entry.getField("volume"));
+            // FIXME BibtetEntryType.getType() ALL_TYPES.get() returns null
+            assertEquals(BibtexEntryTypes.ARTICLE.getName(), entry.getType().getName());
+
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+    @Test
+    public void testImportEntries2() {
+
+        InputStream inStream = InspecImportTest.class.getResourceAsStream("InspecImportTest3.txt");
+        try {
+            List<BibtexEntry> entries = inspecImp.importEntries(InspecImportTest.class
+                    .getResourceAsStream("InspecImportTest3.txt"), new OutputPrinterToNull());
+            assertEquals(1, entries.size());
+            BibtexEntry entry = entries.get(0);
+            // FIXME BibtetEntryType.getType() ALL_TYPES.get() returns null
+            assertEquals(BibtexEntryTypes.INPROCEEDINGS, entry.getType());
+
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+
+    }
+    @Test
+    public void testImportEntries3() {
+
+        InputStream inStream = InspecImportTest.class.getResourceAsStream("InspecImportTest4.txt");
+        try {
+            List<BibtexEntry> entries = inspecImp.importEntries(InspecImportTest.class
+                    .getResourceAsStream("InspecImportTest4.txt"), new OutputPrinterToNull());
+            assertEquals(1, entries.size());
+            BibtexEntry entry = entries.get(0);
+            assertEquals(BibtexEntryTypes.OTHER, entry.getType());
+
+
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
just annotate the testImportEntries3 with throws IOException. We want to avoid e.printStackTrace() at all cost as it is evil. ;)